sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Release History
 
-## 1.14.1 (Unreleased)
+## 1.15.0b1 (Unreleased)
 
 ### Features Added
 
+- Added Windows Web Account Manager (WAM) Brokered Authentication support.
+- Added `enable_msa_passthrough` suppport for `InteractiveBrowserCredential`.
```

Comment:
Could expand this feature description, similar to this comment: https://github.com/Azure/azure-cli/pull/24181